wiki/Documentation-Folder-structure.md

```diff
@@ -1,37 +1,49 @@
 # General
-There is a clear separation of the server side of the application and the client side. Therefore, next to the gitignore and the Readme file there are only two folders in the top structure:
-* One client folder with its own package.json. There you can find everything regarding the React frontend.
-* One server folder with its own package.json. There you can find everything regarding the Node backend and the communication with the database.
+
+There is a clear separation of the server side of the application and the client side (`frontend`). Therefore, next to the README and some config files, there are only four folders in the top structure:
+
+- A **frontend/** folder with its own package.json. There you can find everything regarding the React frontend.
+- A **server/** folder with its own package.json. There you can find everything regarding the Node backend and the communication with the database.
+- A **wiki/** folder which contains all wiki-documents as Markdown file. The pages can be edited in this folder and will be deployed to the wiki on merge to the `main` branch.
```

Comment:
Is it on merge in main or dev? I think on dev, but i'm not sure

Reply:
I also thought it was `DEV`, but in the workflow I saw this:
![image](https://user-images.githubusercontent.com/36270527/118524404-6a0ca100-b73e-11eb-87b7-028a319698b1.png)